src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -162,13 +162,19 @@ private async Task<PullRequestModel> GetPullRequestModel(int prNumber, string re
             if (pullRequestModel == null)
             {
                 string[] repoInfo = repoName.Split("/");
-                var issue = await _githubClient.Issue.Get(repoInfo[0], repoInfo[1], prNumber);
+                var pullRequest = await _githubClient.PullRequest.Get(repoInfo[0], repoInfo[1], prNumber);
+                if(pullRequest.Draft == true)

```

Comment:
How do we catch new APIs when the shift out of draft mode but no CI is triggered?

Reply:
I have reverted this change and updated PR to create consolidated comment for all packages